fluent-plugin-google-cloud.gemspec

```diff
@@ -19,14 +19,14 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'fluentd', '1.11.2'
+  gem.add_runtime_dependency 'fluentd', '~> 1.12', '>= 1.12.2', '< 1.13'
   gem.add_runtime_dependency 'googleapis-common-protos', '1.3.10'
   gem.add_runtime_dependency 'googleauth', '0.9.0'
   gem.add_runtime_dependency 'google-api-client', '0.30.8'
   gem.add_runtime_dependency 'google-cloud-logging', '1.6.6'
   gem.add_runtime_dependency 'google-protobuf', '3.15.8'
   gem.add_runtime_dependency 'grpc', '1.31.1'
-  gem.add_runtime_dependency 'json', '2.2.0'
+  gem.add_runtime_dependency 'json', '~> 2.3.0'
```

Comment:
Which exact version of `json` are we looking for here?

Reply:
We are looking for any version above 2.3.0 which is not vulnerable. We can try `2.4.1` as that is recommended in the fluentd provided docker images?

Reply:
Information here: https://github.com/fluent/fluentd-docker-image/blob/master/v1.12/debian/Dockerfile